oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -177,6 +177,13 @@ When proposing a removal, consider the following analysis:
 Timeline
 --------
 
+To communicate a proposed removal, you will need to determine the *earliest*
+Open edX named release that could no longer have the removed functionality.
+Choosing the earliest named release where removal could be complete is the
```

Comment:
Minor quibble on the wording: The consumers of DEPR announcements are often less interested in the earliest date that the feature could be *completely* removed (all code and associated resources deleted, etc) and more interested in the earliest date that any bit of it could stop working.
```suggestion
Choosing the earliest named release where functionality could be removed
```